Utilities/Pipelines/Scripts/fetchTools.cmd

```diff
@@ -0,0 +1,19 @@
+echo Running fetchTools.cmd
+
+rem Log build machine state
+if EXIST "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\amd64\msbuild.exe" "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\amd64\msbuild.exe" /version
+if EXIST "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\amd64\MSBuild.exe" "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\amd64\MSBuild.exe" /version
+dir /b "C:\Program Files (x86)\Microsoft SDKs"
+dir /b "C:\Program Files (x86)\Windows Kits\10\Include"
+set
+
+set patArg=%1
+set toolsUrl=%2
+cd /D %BUILD_STAGINGDIRECTORY%
+call git clone https://anything:%patArg%@%toolsUrl%
```

Comment:
interesting - how does this `anything` work?

Reply:
its just how git works:

[https://github.com/MicrosoftDocs/azure-devops-docs/issues/2455#issuecomment-439503194](https://github.com/MicrosoftDocs/azure-devops-docs/issues/2455#issuecomment-439503194)

Reply:
oh i see it's the username for the PAT